netutils/ping/icmpv6_ping.c

```diff
@@ -289,7 +308,7 @@ void icmp6_ping(FAR const struct ping6_info_s *info)
               goto done;
             }
 
-          elapsed = (unsigned int)TICK2MSEC(clock() - start);
+          elapsed = (unsigned int)TICK2USEC(clock() - start);
```

Comment:
why do we need to switch from `TICK2MSEC` to `TICK2USEC`?

Reply:
The return value of the ping command is accurate to milliseconds

Reply:
So what was the problem of using time base on milliseconds? I mean why we change from milliseconds to microseconds?

Reply:
Statistics are performed in milliseconds. The time value in the returned packet of the ping command cannot be accurate to two decimal places.